Framework/CurveFitting/src/Algorithms/Fit.cpp

```diff
@@ -48,6 +48,9 @@ void Fit::initConcrete() {
       ->setDocumentation("Whether the fit was successful");
   declareProperty("OutputChi2overDoF", 0.0, "Returns the goodness of the fit",
                   Kernel::Direction::Output);
+  declareProperty("OutputNIterations", 0, Kernel::Direction::Output);
```

Comment:
This is done for performance reasons: if you don't need the table workspace with the parameters or the covariance matrix you may choose not to create them. You can access the optimised parameters via Function itself which is an `InOut` property. How does `OutputNIterations` break the python interface?